### PR TITLE
[flutter_markdown] fix for latest pkg:markdown part 2

### PR DESCRIPTION
**Pull request by chenzhiguang**
The same reason as this PR: https://github.com/flutter/packages/pull/2777

CC @srawlins @kevmoo

### CODE REVIEW

File: packages/flutter_markdown/pubspec.yaml [hunk 1]
```diff
@@ -7,8 +7,8 @@ issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+
 version: 0.6.13
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
```
Comment (kevmoo) [inline packages/flutter_markdown/pubspec.yaml:10]: revert these changes. Good to avoid anything unrelated!

File: packages/flutter_markdown/test/link_test.dart [hunk 2]
```diff
@@ -6,8 +6,12 @@ import 'package:flutter/gestures.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_markdown/flutter_markdown.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:markdown/markdown.dart' as md show version;
 import 'utils.dart';
 
+// TODO(Zhiguang): delete this once the min version of pkg:markdown is updated
+final bool _newMarkdown = md.version.compareTo('6.0.1') > 0;
```
Comment (kevmoo) [inline packages/flutter_markdown/test/link_test.dart:13]: We have this in another test file, right? Maybe we should make it public and put it in a shared test file?

**Review by kevmoo — APPROVED** [review]
Awesome!

File: packages/flutter_markdown/pubspec.yaml (hunk 1 above)
Comment (chenzhiguang) [reply packages/flutter_markdown/pubspec.yaml]: Yep, I found it too. Thanks!

File: packages/flutter_markdown/pubspec.yaml [hunk 3]
```diff
@@ -20,4 +20,4 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  mockito: ^5.0.0
+  mockito: ^5.0.0
```
Comment (kevmoo) [inline packages/flutter_markdown/pubspec.yaml:23]: oops! newline!

File: packages/flutter_markdown/test/link_test.dart (hunk 2 above)
Comment (chenzhiguang) [reply packages/flutter_markdown/test/link_test.dart]: Agreed, fixed!

**Review by kevmoo — APPROVED** [review]
